static/js/actions/profile.js

```diff
@@ -64,6 +64,16 @@ export const updateProfileValidation = createAction(UPDATE_PROFILE_VALIDATION,
   (username, errors) => ({ errors, username })
 );
 
+export const UPDATE_VALIDATION_VISIBILITY = 'UPDATE_VALIDATION_VISIBILITY';
+export const updateValidationVisibility = createAction(UPDATE_VALIDATION_VISIBILITY,
+  (username, keySet) => ({ username, keySet })
+);
+
+export const CLEAR_VALIDATION_VISIBILITY = 'CLEAR_VALIDATION_VISIBILITY';
+export const clearValidationVisibility = createAction(CLEAR_VALIDATION_VISIBILITY,
```

Comment:
My IDE says this isn't being used anywhere, and the reducer for this doesn't have any coverage when running tests. Should we delete this?